tawazi/config.py

```diff
@@ -18,6 +21,9 @@ class Config(BaseSettings):
     # Weather to run the graphs in debug mode or not
     RUN_DEBUG_NODES: bool = False
 
+    # Default timeout for all nodes
+    TAWAZI_DEFAULT_TIMEOUT: Optional[float] = None
```

Comment:
None can be omitted, but really that's ok to leave it

Reply:
I prefer to leave it for clarification only